scripts/apps/archive/directives/RelatedItemsPreview.ts

```diff
@@ -16,6 +16,7 @@ export function RelatedItemsPreview(relationsService) {
         scope: {
             item: '=',
             field: '<',
+            showActionMenu: '=?',
```

Comment:
when should it not be shown?

Reply:
When you open the preview of an article.

Reply:
why not show it in the preview?

Reply:
Well, I did ask Karel, and he said:
> nope, the preview is a preview. I think there is no need to have the actions there

Reply:
Can you, in this case, pass a `preview: boolean` instead of `showActionMenu` and hide the menu based on that? It's easier to understand the reasoning this way without digging too much into the code.